Installation/cmake/modules/CGALConfig_binary.cmake.in

```diff
@@ -186,7 +186,9 @@ include(${CGAL_MODULES_DIR}/CGAL_Macros.cmake)
 # Temporary? Change the CMAKE module path
 cgal_setup_module_path()
 
-if( RUNNING_CGAL_AUTO_TEST )
+if( CGAL_DEV_MODE OR RUNNING_CGAL_AUTO_TEST )
+  # Do not use -isystem for CGAL include paths
+  set(CMAKE_NO_SYSTEM_FROM_IMPORTED TRUE)
```

Comment:
I think that is a bad solution, and the cause of all the warnings from third-party libraries we use.

Instead, I should have set the `NO_SYSTEM_FROM_IMPORTED` on the CGAL exported targets. -->TODO for myself.